spanish-terminology.md

```diff
@@ -11,14 +11,14 @@
 | Keys | Claves | Se discutió "llaves", pero la comunidad Bitcoin ya utiliza la palabra "claves" |
 | Mine | Minar | |
 | Miner | Minero o Minador| |
-| Mnemonic seed | Semilla mnemotécnica | Semilla "mnemónica" es un argot, jerga. La palabra mnemotécnica viene de las siguientes palabras "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
+| Mnemonic seed | Semilla mnemómica | Mnemónica,co [RAE](http://dle.rae.es/?id=PRbUjYS) "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
```

Comment:
ojo al error de ortografía: pusiste "mnemómica"

La DRAE dice en cuanto a "mnemotécnica":
  "2. adj. Que sirve para auxiliar a la memoria."
lo que es la función de "mnemonic seed", 

en el caso de "mnemónica", no menciona ese significado.

Reply:
Yes it does, go to this link [RAE mnemónico/a](http://dle.rae.es/?id=PRbUjYS)

Reply:
btw, thank you for seeing the spelling error. Missed it while writing to fast, I'll be more careful in the future, thank you :).